context/app/static/js/components/Detail/ProvAnalysisDetailsLink/ProvAnalysisDetailsLink.jsx

```diff
@@ -6,7 +6,7 @@ import { CwlIcon, FlexLightBlueLink, PrimaryTextDivider, StyledListItem } from '
 function ProvAnalysisDetailsLink(props) {
   const { data } = props;
 
-  const trimmedOrigin = data.origin.split('.').slice(0, -1).join('.');
+  const trimmedOrigin = data.origin.endsWith('.git') ? data.origin.split('.').slice(0, -1).join('.') : data.origin;
```

Comment:
```suggestion
  const trimmedOrigin = data.origin.replace(/\.git$/, '');
```
I think this would be more clear.